File: CS/MyRadioGroup/MyRadioGroupViewInfo.cs
using System;
using System.Drawing;
using System.Collections.Generic;
using DevExpress.XtraEditors.Repository;
using DevExpress.XtraEditors.ViewInfo;
using DevExpress.Utils;



namespace DevExpress.MyControl
{
    public class MyRadioGroupViewInfo : RadioGroupViewInfo
    {
        public MyRadioGroupViewInfo(RepositoryItem item)
            : base(item)
        {
        }


        public override ToolTipControlInfo GetToolTipInfo(Point point)
        {
            ToolTipControlInfo resInfo = base.GetToolTipInfo(point);

            RepositoryItemMyRadioGroup ri = Item as RepositoryItemMyRadioGroup;
            if (ri.OwnerEdit != null)
            {
                for (int itemIndex = 0; itemIndex < Item.Items.Count; itemIndex++)
                {
                    Rectangle rect = ri.OwnerEdit.GetItemRectangle(itemIndex);
                    if (rect.Contains(point) && ri.Items[itemIndex].ToolTip != string.Empty)
                    {
                        resInfo = null;
                        resInfo = new ToolTipControlInfo(itemIndex, ri.Items[itemIndex].ToolTip);
                    }
                }
            }

            return resInfo;
        }
    }
}

File: VB/TestMyRadioGroup/MyRadioGroupViewInfo.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Drawing
Imports System.Collections.Generic
Imports DevExpress.XtraEditors.Repository
Imports DevExpress.XtraEditors.ViewInfo
Imports DevExpress.Utils



Namespace TestMyRadioGroup
	Public Class MyRadioGroupViewInfo
		Inherits RadioGroupViewInfo
		Public Sub New(ByVal item As RepositoryItem)
			MyBase.New(item)
		End Sub


		Public Overrides Function GetToolTipInfo(ByVal point As Point) As ToolTipControlInfo
			Dim resInfo As ToolTipControlInfo = MyBase.GetToolTipInfo(point)

			Dim ri As RepositoryItemMyRadioGroup = TryCast(Item, RepositoryItemMyRadioGroup)
			If ri.OwnerEdit IsNot Nothing Then
				For itemIndex As Integer = 0 To Item.Items.Count - 1
					Dim rect As Rectangle = ri.OwnerEdit.GetItemRectangle(itemIndex)
					If rect.Contains(point) AndAlso ri.Items(itemIndex).ToolTip <> String.Empty Then
						resInfo = Nothing
						resInfo = New ToolTipControlInfo(itemIndex, ri.Items(itemIndex).ToolTip)
					End If
				Next itemIndex
			End If

			Return resInfo
		End Function
	End Class
End Namespace
